### PR TITLE
Google Cloud refactor, GCR and GCE separation, weighting fixes

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _pro/continuous-deployment/google-cloud.md [hunk 1]
```diff
@@ -23,152 +23,126 @@ You can find a sample repo for deploying to Google Cloud with Codeship Pro on Gi
...
 
-echo "Listing services on GCE"
-kubectl get services $KUBERNETES_APP_NAME
+- We authenticate with Google Cloud
+- Then, we are setting the default zone to use
+- Next, we are starting an instance in the Google Compute Engine.
 
-echo "Removing service $KUBERNETES_APP_NAME"
-kubectl delete services $KUBERNETES_APP_NAME
+As you can see, the deployment script is essentially just standard Google Cloud CLI commands, meaning you can any Google Cloud commands that you want.
```
Comment (mlocher) [inline _pro/continuous-deployment/google-cloud.md]: there's a verb missing in the second part of the sentence.

File: _pro/continuous-deployment/google-cloud.md [hunk 2]
```diff
@@ -23,152 +23,126 @@ You can find a sample repo for deploying to Google Cloud with Codeship Pro on Gi
...
 
-echo "Waiting After Remove"
+You can also take a look at a [longer example we use for integration testing our container](https://github.com/codeship-library/google-cloud-deployment/blob/master/test/deploy_to_google.sh).
```
Comment (mlocher) [inline _pro/continuous-deployment/google-cloud.md]: The URL is broken, the working URL from the repository is https://github.com/codeship-library/google-cloud-deployment/blob/master/deployment/test/deploy_to_google.sh

File: _pro/continuous-deployment/google-cloud.md [hunk 3]
```diff
@@ -23,152 +23,126 @@ You can find a sample repo for deploying to Google Cloud with Codeship Pro on Gi
...
 
-echo "Stopping port forwarding for $KUBERNETES_APP_NAME"
-kubectl stop rc $KUBERNETES_APP_NAME
+## Container Engine And Container Registry
 
-echo "Stopping Container Cluster for $KUBERNETES_APP_NAME"
-gcloud container clusters delete $KUBERNETES_APP_NAME -q
-```
+If you are looking to use Google Container Engine and Google Container registry, we main [specific documentation]({{ site.baseurl }}{% link _pro//continuous-deployment/google-container.md %}) for using those services.
```
Comment (mlocher) [inline _pro/continuous-deployment/google-cloud.md]: main should probably be _maintain_

File: _pro/continuous-deployment/google-container.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+Your new environment variables file will container the following:
+
+```bash
+GOOGLE_AUTH_JSON=...
+GOOGLE_AUTH_EMAIL=...
+GOOGLE_PROJECT_ID=...
+```
+
+- `GOOGLE_AUTH_JSON` should be populated with the account credential string you received in the json file you downloaded earlier.  **Note** that you will need to remove all newlines from the file. On Linux and OSX you can use `tr '\n' ' ' < your_file_name` to get the line and copy it back into the file. You can find the
```
Comment (mlocher) [inline _pro/continuous-deployment/google-container.md]: second half of the sentence is missing.

**Comment by ethangj** [comment]
@mlocher Updated

**Pull request merged**